src/concurrency.md

```diff
@@ -72,6 +73,46 @@ fn main() {
 }
 ```
 
+[ex-rayon-parallel-search]: #ex-rayon-parallel-search
+<a name="ex-rayon-parallel-search"></a>
+## Search unordered items in parallel
+
+[![rayon-badge]][rayon] [![cat-concurrency-badge]][cat-concurrency]
+
+This example uses [`rayon::find_any`] and [`par_iter`] to search a vector in
+parallel for an element satisfying the predicate in the given closure.
+
+If there are multiple elements satisfying the predicate defined in the closure
+argument of [`rayon::find_any`], we are only guaranteed that one of them will be
+found, but not necessarily that the first will be found.
+
+Also note that the argument to the closure is a reference to a reference
+(`&&x`). Please see the discussion on [`std::find`] for additional details.
+
+```rust
+extern crate rayon;
+
+use rayon::prelude::*;
+
+fn main() {
+    let v = vec![6, 2, 1, 9, 3, 8, 11];
+
+    let f1 = v.par_iter().find_any(|&&x| x == 9);
+    let f2 = v.par_iter().find_any(|&&x| x % 2 == 0 && x > 6);
+    let f3 = v.par_iter().find_any(|&&x| x > 8);
+
+    if let Some(found_9) = f1 {
```

Comment:
How about writing it more concisely as
```rust
    assert_eq!(f1, Some(&9));
    assert_eq!(f2, Some(&8));
    assert!(f3 > Some(&8))
```

Reply:
This is much cleaner. It did not occur to me to convert the expected value into an `optional`. Very clever! 👍